pykern/pkcli/ci.py

```diff
@@ -147,3 +136,15 @@ def _paths(cwd):
         _error("no files found")
     if r:
         _error("\n".join(r))
+
+
+def _paths(cwd):
+    if pkio.py_path("setup.py").isfile() and (
+        pkio.py_path("README.rst").isfile() or pkio.py_path("README.md").isfile()
+    ):
+        return (
+            cwd.basename,
```

Comment:
This line needs to be documented. This is a the repo name, but that's a posit based on how we setup repos.

Reply:
3b6bc4ce71f665b83fc9f9cf3024c5434a631d09